dart/lib/src/sentry_client.dart

```diff
@@ -99,6 +100,15 @@ class SentryClient {
       event = event.copyWith(exception: sentryException);
     }
 
+    if (_options.attachStackTrace &&
```

Comment:
should we test if event.stackTrace is `null` ?

Reply:
yes only if its null, but actually I'd do this as `else if` with the exceptions check above, its either the exception with a stacktrace or a threads.stacktrace, not both, this way makes possible that we have a exception.stacktrace and still a threads.thread.stacktrace which is wrong.

Reply:
✅